bin/MadGraph5_aMCatNLO/cards/production/13TeV/higgs/ggh01_M125_Toa01a01_M15_Tomumubb/ggh01_M125_Toa01a01_M15_Tomumubb_run_card.dat

```diff
@@ -265,7 +265,9 @@
 #          meaningful ONLY if plotted taking matchscale              *
 #          reweighting into account!                                 *
 #*********************************************************************
-   F  = use_syst      ! Enable systematics studies
+#   F  = use_syst      ! Enable systematics studies
+   F  = cut_decays      ! Enable systematics studies   
```

Comment:
you already have 
T  = cut_decays
in line 92, could you please remove this and modify above?
Also, instead of commenting F  = use_syst and duplicating the line,
I suggest to have a single line with
T = use_syst

This comments apply also to the other files.

Thanks,
Luca